Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/ContactDiaryActivity.kt

```diff
@@ -37,7 +37,7 @@ class ContactDiaryActivity : AppCompatActivity(), HasAndroidInjector {
         val navInflater = navController.navInflater
         val graph = navInflater.inflate(R.navigation.contact_diary_nav_graph)
 
-        if (settings.isOnboarded.value) {
+        if (settings.isOnboarded.value && settings.isExtendedOnboarded.value) {
```

Comment:
Although this works nicely (tested on Pixel with Android 10), I'd suggest to persist and check for the type of onboarding that has been shown yet, instead of an additional boolean value. E.g. an enum with 

```
enum classCjOnboardingShown(val order: Int) {
   RISKSTATUS_1_12(1),
   FUTURE_CHANGE_1_13(2)
}
```

So in the future, we can check if we have shown the latest onboarding yet or not, without having to chain several boolean values.

Reply:
This is a good suggestion.

Reply:
changed as suggested